recipes/snakebite-py3/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "snakebite-py3" %}
+{% set version = "3.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/snakebite-py3-{{ version }}.tar.gz
+  sha256: 2026f2f755b63503145b6ab3037ca2b5905c9d1c68d3e7903b90ab94ac7fa260
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - protobuf >2.4.1
```

Comment:
is this a > instead of >= ?

Reply:
I agree, it's odd, but that's what upstream has:
https://github.com/internetarchive/snakebite-py3/blob/master/setup.py#L45

Reply:
/shrug oh well